### ISSUE TITLE
<br> breaks form layout which is inside a horizontal layout

### ISSUE THREAD

**Issue by haiwan** [vaadin/vaadin-form-layout#78]
Code below breaks the form layout which is insde a horizontal layout. Without the br element, everything works fine, after adding the br, UI is broken. See the screenshots
<img width="1541" alt="with-br" src="https://user-images.githubusercontent.com/3388675/40496619-e0749b2a-5f82-11e8-8640-f1f87b2deff2.png">
<img width="1541" alt="without-br" src="https://user-images.githubusercontent.com/3388675/40496620-e097f12e-5f82-11e8-97e1-dff9078e021f.png">

```
<head>
<link rel="import" href="bower_components/polymer/polymer-element.html">
<link rel="import" href="bower_components/vaadin-ordered-layout/vaadin-horizontal-layout.html">
<link rel="import" href="bower_components/vaadin-form-layout/vaadin-form-layout.html">
<link rel="import" href="bower_components/vaadin-form-layout/vaadin-form-item.html">
</head>

<body>
  <vaadin-horizontal-layout align-items="stretch" style="width: 100%; height:100%">
    <div style="width: 20%">I'm a menu bar</div>

    <vaadin-form-layout style="flex-grow: 1">

      <vaadin-form-item>
        <label slot="label">Email</label>
        <input value="jane.doe@example.com">
      </vaadin-form-item>
      <br>
      <vaadin-form-item>
        <label slot="label">Confirm Email</label>
        <input value="jane.doe@example.com">
      </vaadin-form-item>

    </vaadin-form-layout>
  </vaadin-horizontal-layout>
</body>

```

**Comment by jouni** [comment]
This works as expected and as designed. You can use the `<br>` element to force a new row on the form layout.

**Comment by jouni** [comment]
Ah, right! Thanks for the clarification, missed the scrollbar 😄

**Comment by limonte** [comment]
The fix has been released in [v2.0.1](https://github.com/vaadin/vaadin-form-layout/releases/tag/v2.0.1) :tada: 

Thanks for reporting @haiwan!
